modules/virt-configuring-vm-use-usb-device.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-VM-use-usb-device_{context}"]
+= Configuring a virtual machine connection to a USB device
+
+You can configure virtual machine (VM) access to a USB device. Doing so allows a guest to connect to actual USB hardware that is attached to an {product-title} node, as if the hardware and VM are physically connected.
+
+.Prerequisites
+
+* A cluster admin must enable the `hostDevices` feature gate on the HyperConverged (HCO) custom resource (CR).
+
```

Comment:
@tiraboschi Is this correct? Do we have API for USBs?

Reply:
No, still not.

Reply:
@acardace, from your comment in Slack that "the feature is now ready, there were some technical things missing but nothing changes in terms of docs" I'm thinking that the API for USBs is now available and the doc can stay as is. If that's not right, please let me know.